VeterinaryVladimir/Program.cs

```diff
@@ -8,8 +8,231 @@ namespace VeterinaryVladimir
 {
     class Program
     {
+        //Использовать набор "домашних животных" классов для написания программы для работы с животными в клинике в отделении "стационара.
+        //Программа должна "висеть" в главном меню:
+        //    [H] About company
+        //    [R] Put new animal in clinic
+        //    [L] Show list of animals
+        //    [D] Discharge animal
+        //    [Q] Quit
+        //    (введена ли прописная буква или строчная, не имеет значения).
+        //"Put new animal in clinic" значит, что клиент принес животное, которое регистрируется в клинике как лечащееся.
+        //   Животному присваивается индивидуальный номер, абсолютно уникальный среди других когда-либо лечившихся животных.
+        //"Show list of animals" показывает список животных, которые сейчас лечатся в клинике.
+        //"Discharge animal" значит, что животное выписывается. Животных выписывают по индивидуальному номеру,
+        //    при выборе этого пункта программа должна спросить номер животного.
+
+
         static void Main(string[] args)
         {
+            ConsoleKeyInfo cki;
+            Clinica clinica = new Clinica();
+
+            HomeDog enimal1 = new HomeDog("Тобик", 3);
+            HomeDog enimal2 = new HomeDog("Бобик", 3);
+            HomeDog enimal3 = new HomeDog("Бибик", 3);
+
+            HomeCat enimal4 = new HomeCat("Соня", 4);
+            HomeCat enimal5 = new HomeCat("Моня", 4);
+            HomeCat enimal6 = new HomeCat("Фоня", 4);
+            /*
+               HomeHamster enimal7  = new HomeHamster("Барон", 3);
+               HomeHamster enimal8  = new HomeHamster("Банан", 3);
+               HomeHamster enimal9  = new HomeHamster("БоБо" , 3);
+               
+               HomeFish enimal10 = new HomeFish("Золото",3);
+               HomeFish enimal11 = new HomeFish(""      ,3);
+               HomeFish enimal12 = new HomeFish(""      ,3);
+            */
+
+            clinica.addAnimal(enimal1);
+            clinica.addAnimal(enimal2);
+            clinica.addAnimal(enimal3);
+
+            clinica.addAnimal(enimal4);
+            clinica.addAnimal(enimal5);
+            clinica.addAnimal(enimal6);
+
+
+            for (int animalIndex = 0; animalIndex < clinica.getAnimals().Count; animalIndex++)
```

Comment:
If you implement IEnumerable<CartAnimal> in Clinica class, you will be able to write this way:
foreach(var animal in clinica)
which seems to be more readable and more effective because you will get animal reference without additional index access.